dagmc_stats/DagmcQuery.py

```diff
@@ -77,6 +80,34 @@ def __get_entities(self, meshset):
                             'volumes. Rootset will be used by default.')
             self.meshset_lst = self.dagmc_file.entityset_ranges['surfaces']
 
+    def __get_volumes(self, meshset):
+        """convert the list of meshsets to its corresponding list of volumes
+
+        inputs
+        ------
+            meshset: the list of meshsets on which _vol_data query will be performed.
+
+        outputs
+        -------
+            none
+        """
+        if type(meshset) != list:
+            # convert single item to list for iterating
+            meshset = [meshset]
+        for m in meshset:
+            dim = self.dagmc_file._my_moab_core.tag_get_data(
+                self.dagmc_file.dagmc_tags['geom_dim'], m)[0][0]
+            # get volumes
+            if dim == 3:
+                self.vols.append(m)
+
+        self.vols = list(set(self.vols))
+        # if no items in the meshset list is a volume,
+        # then use rootset by default instead
+        if len(self.meshset_lst) == 0:
+            warnings.warn('Specified meshset(s) are not volumes. Rootset will be used by default.')
+            self.vols = self.dagmc_file.entityset_ranges['volumes']
```

Comment:
This is now more complicated.... if the meshset only contains surfaces, then we should leave this empty.  We don't want the list of volumes to be inconsistent with the list of surfaces.